src/components/Login/Login.jsx

```diff
@@ -0,0 +1,81 @@
+import { React, useState } from 'react';
+import { Link, useHistory } from 'react-router-dom';
+import Form from '../UI/Form/Form';
+import Button from '../UI/Button/Button';
+import Input from '../UI/Input/Input';
+import { FetchSend } from '../../utils/utils.js';
+import Error from '../UI/Error/Error';
+
+const Login = ({ updateUserName, messageForm, updateMessageForm }) => {
+	const router = useHistory();
+	const [userData, setUserData] = useState({
+		email: '',
+		password: '',
+	});
+
+	const handlerUserData = (event) => {
+		setUserData({
+			...userData,
+			[event.target.name]: event.target.value,
+		});
+	};
```

Comment:
Same:
```suggestion
	const handlerUserData = (target: { name, value }) => {
		setUserData({
			...userData,
			[name]: value,
		});
	};
```
Would be nice to add some validation here.
User name and password shouldn't be empty, at least 3 symbols in name, at least 6 symbols in password.